mne/io/cnt/tests/test_cnt.py

```diff
@@ -0,0 +1,28 @@
+
+# Author: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+from nose.tools import assert_equal
+
+import mne
+from mne.utils import run_tests_if_main
+from mne.datasets import testing
+from mne.io.tests.test_raw import _test_raw_reader
+from mne.io.cnt import read_raw_cnt
+
+data_path = testing.data_path(download=True)
+fname = op.join(data_path, 'CNT', 'scan41_short.cnt')
```

Comment:
can we avoid capital letters in file names? yes some OS are still not case sensitive...